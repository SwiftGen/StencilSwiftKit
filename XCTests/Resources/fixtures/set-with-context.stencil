// before set
x = "{{x}}"
y = "{{y}}"

// set value
{% set x %}hello{% endset %}
{% set y %}world{% endset %}
x = "{{x}}"
y = "{{y}}"

// set inside for loop
{% for item in items %}
  {% set x %}item #{{item}}{% endset %}
  x = "{{x}}"
  y = "{{y}}"
{% endfor %}

// after for loop
x = "{{x}}"
y = "{{y}}"
