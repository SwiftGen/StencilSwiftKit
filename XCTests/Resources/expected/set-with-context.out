// before set
x = "1"
y = "2"

// set value
x = "hello"
y = "world"

// set inside for loop
  x = "item #1"
  y = "world"
  x = "item #2"
  y = "world"
  x = "item #3"
  y = "world"

// after for loop
x = "hello"
y = "world"
